### PR TITLE
Prepare version 0.33.0

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR closes https://github.com/Ouranosinc/xclim/milestone/28
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)

### What kind of change does this PR introduce?

* Prepare version 0.33.0

### Does this PR introduce a breaking change?

This will be the last version to support Python==3.7 and xarray<0.21. 

### Other information:

### CODE REVIEW

**Comment by Zeitsperre** [comment]
Waiting on merge of #998 before merging this.

**Review by aulemahal — APPROVED** [review]

**Pull request merged**
